eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/PartitionProperties.java

```diff
@@ -14,27 +14,33 @@
     private final long beginningSequenceNumber;
     private final long lastEnqueuedSequenceNumber;
     private final String lastEnqueuedOffset;
-    private final Instant lastEnqueuedTimeUtc;
+    private final Instant lastEnqueuedTime;
     private final boolean isEmpty;
-    private Instant propertyRetrievalTimeUtc;
+    private Instant propertyRetrievalTime;
 
-    PartitionProperties(
-            final String eventHubPath,
-            final String id,
-            final long beginningSequenceNumber,
-            final long lastEnqueuedSequenceNumber,
-            final String lastEnqueuedOffset,
-            final Instant lastEnqueuedTimeUtc,
-            final boolean isEmpty,
-            final Instant propertyRetrievalTimeUtc) {
+    /**
+     * Creates an instance with all of the partition's properties set.
+     *
+     * @param eventHubPath Name of the Event Hub this partition belongs to.
+     * @param id Identifier of the partition, unique to the Event Hub which contains it.
+     * @param beginningSequenceNumber The first sequence number available for events in the partition.
+     * @param lastEnqueuedSequenceNumber The sequence number of the most recent event enqueued to this partition.
+     * @param lastEnqueuedOffset The offset of the most recent event enqueued to this partition.
+     * @param lastEnqueuedTime The date time (UTC) of the most recent event enqueued to this partition.
+     * @param isEmpty {@code true} if there are no events in the partition; {@code false} otherwise.
+     * @param propertyRetrievalTime A date time (UTC) representing when the partition's properties were retrieved.
+     */
+    public PartitionProperties(String eventHubPath, String id, long beginningSequenceNumber,
+                               long lastEnqueuedSequenceNumber, String lastEnqueuedOffset, Instant lastEnqueuedTime,
+                               boolean isEmpty, Instant propertyRetrievalTime) {
```

Comment:
Does there need to be a public ctor here? I would have assumed we would be the only ones creating instances (which would be nice, because the parameter count here is crazy). I see this is instantiated from implementation classes, so perhaps we need to do a little more work to prevent the need for this to be public API.

Reply:
Fixed